src/utils.ts

```diff
@@ -59,6 +59,17 @@ export function parseParameters(url: string) {
     return mapEquivalentParams(params);
 }
 
+function generateHeaders(device: 'mobile' | 'desktop') {
+    const headerGenerator = new HeaderGenerator({
+        devices: [device],
+    });
+    const generatedHeaders = headerGenerator.getHeaders();
+    // remove 'te' header as it is causing page.goto: net::ERR_INVALID_ARGUMENT error
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { te, ...rest } = generatedHeaders;
```

Comment:
Can you please always try to report these things to #bugs, ideally with some simple reproduction. We need to keep fixing whatever tooling issues we have even that it costs us some time. Thanks